core/src/main/java/org/elasticsearch/cluster/metadata/IndexTemplateMetaData.java

```diff
@@ -84,6 +111,16 @@ public int getOrder() {
         return order();
     }
 
+    @Nullable
+    public Integer getVersion() {
+        return version();
+    }
+
+    @Nullable
+    public Integer version() {
```

Comment:
I added them for consistency because every other method has both. I can remove them, but I think we should keep them both until we go back and remove _all_ of the dupes.

Frankly, we need to rewrite the whole template pathway because it has way too many indirection layers, which has led this weirdness.